README.md

```diff
@@ -49,6 +49,7 @@ xcov -w LystSDK.xcworkspace -s LystSDK -o xcov_output
 * `--include_test_targets`: Enables coverage reports for `.xctest` targets.
 * `--ignore_file_path` `-x`: Relative or absolute path to the file containing the list of ignored files.
 * `--exclude_targets`: Comma separated list of targets to exclude from coverage report.
+* `--include_targets`: Comma separated list of targets to include in coverage report.
```

Comment:
Shall we add a comment specifying that "if defined, `exlude_targets` will be ignored"?